service_mesh/v2x/ossm-performance-scalability.adoc

```diff
@@ -1,13 +1,15 @@
 [id="ossm-performance-scalability"]
 = Performance and scalability
+include::modules/ossm-document-attributes.adoc[]
 :context: performance-scalability
 
 toc::[]
 
-The default `ServiceMeshControlPlane` settings are not intended for production use; they are designed to install successfully on a default OpenShift installation, which is an extremely resource limited environment. After you have verified a successful SMCP installation, you should modify the settings defined within the SMCP to suit your environment. This section provides guidance for configuring {ProductName} for production use.
+The default `ServiceMeshControlPlane` settings are not intended for production use; they are designed to install successfully on a default OpenShift installation, which is a resource limited environment. After you have verified a successful SMCP installation, you should modify the settings defined within the SMCP to suit your environment.
```

Comment:
I know this is existing text, but do you want to:
s/OpenShift/{product-title}
s/resource limited/resource-limited

Reply:
fixed